simulator-core/src/test/java/com/evbox/everon/ocpp/simulator/station/handlers/ocpp/CertificateSignedRequestHandlerTest.java

```diff
@@ -0,0 +1,120 @@
+import com.google.common.base.Splitter;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import static com.evbox.everon.ocpp.mock.constants.StationConstants.DEFAULT_MESSAGE_ID;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.Mockito.verify;
+
+@ExtendWith(MockitoExtension.class)
+public class CertificateSignedRequestHandlerTest {
+
+    @Mock
+    StationStore stationStoreMock;
+    @Mock
+    StationMessageSender stationMessageSenderMock;
+    @Captor
+    ArgumentCaptor<CertificateSignedResponse> messageCaptor;
+
+    @InjectMocks
+    CertificateSignedRequestHandler requestHandler;
+
+    private static final String expiredCertificate = "3082035830820240a003020102020500f8b10766300d06092a864886f70d01010b05003075310b3009060355040613024e4c310b300906035504080c024e4c310c300a06035504070c03414d53310e300c060355040a0c055375624341310e300c060355040b0c055375624341310e300c06035504030c055375624341311b301906092a864886f70d010901160c7375626361407375622e6361301e170d3138313130383039323832345a170d3139313130373039323832345a30173115301306035504030c0c30303030303030303030304630820122300d06092a864886f70d01010105000382010f003082010a0282010100dbd865c2c144dbceab605ec2dcec80d8ac5b6e9a3ed98895c569dfffc71b7ab4e074628cf4edfc071ae1bc75f3ce88578719dbd05d8ff27a79883488325d0b7131497b518adf205cd3f16e4698fb116b613f088621eba6c7fd65b9960e882961340bf3882f37f4168a726f1f367821f7856d59658a5508769d8c39401ee349e77edf5a03a1f9b817adc17b7eaa75d560fb3cc0b5e6c3478486e2ae76db3f796be234d6f080937537a61c6a9b85c2ed520028a6c13909f386f3adc726f92fb4c92c8babf3cbc7d7e4c3b1f3b5d54f376726947761ad7aea0d33e6c4537427cf6da12f011b928b119083b6af3b797991ba519a976d22d473d65d84c09a74a722db0203010001a34d304b30090603551d1304023000301d0603551d0e04160414c078da2313232fa6293b4d31253d103f0b856972301f0603551d23041830168014fce4a2ae5d72e6af9f8ada3af4bc15b7668cdeb5300d06092a864886f70d01010b050003820101009522c037a121b020a2f577cd79ce1b77f886c24c9d952895d34c4455d1f6d396b7fc08b9a454eb049fb4e5fab0d1cd45b154ea5390110f4f28a6c6577185bb3b6c651232dc8c942466fce42b7468c2c269b6fc3709e2b7aa4733883e58310ae260112ffd5f69be04cf9fbcfc18daf0b956766486f66f2eabb5d416e7a5ce69d670b2fb2f7fbd456be27fa0bee1def27e7ff4f3c5701cfe8a47fcb48f798f6989529cbf42d09634c5666811ecb4b525d78e40678fe16ac69a478fa5d1a819586c1e7b8b3d1f3802e4857ab419f2310635ebd04ba89317f387b9f02be7fef5bf4c5498132f530fbcf96c8e48a75088dce1122e8f116fa1cc82ed131286867eb8e4\n" +
```

Comment:
GitHub refuses to display diff for this file by default because of long strings. Can we move them to resources somehow?